lang/java/reef-tang/tang/src/main/java/org/apache/reef/tang/ClassHierarchy.java

```diff
@@ -42,7 +42,6 @@
    * @param fullName The full name of the class that will be looked up.
    * @return A non-null reference to a ClassNode or a NamedParameterNode.
    * @throws NameResolutionException If the class is not found.
-   * @throws ClassHierarchyException If the class does not pass Tang's static analysis.
```

Comment:
We no longer throw this? I believe we do, but it is a `RuntimeException` we don't need to declare. Having it in the JavaDoc is useful, though.